### ISSUE TITLE
master doesn't compile - README updates needed

### ISSUE THREAD

**Issue by lawrenceleejr** [rsmith54/RJigsawTools#67]
Out of the box, the current master (07904e40f45456c1286c170f3330d82fa0ffc942) doesn't compile. First we need to add SUSYTools to the README since we've switched to AnalysisBase.

And then in the compilation of xAODAnaHelpers, I get the following error. So there's some incompatibility in the various versions in the README. @rsmith54, I know you have a working setup - can you update the readme to use the tags you're using?

-L

```
/afs/cern.ch/work/l/leejr/public/TwoLeptonFactory/xAODAnaHelpers/Root/BasicEventSelection.cxx: In member function ‘virtual EL::StatusCode BasicEventSelection::execute()’:
/afs/cern.ch/work/l/leejr/public/TwoLeptonFactory/xAODAnaHelpers/Root/BasicEventSelection.cxx:600:39: error: no matching function for call to ‘CP::PileupReweightingTool::apply(const EventInfo&)’ 
       m_pileuptool->apply( *eventInfo ); // NB: this call automatically decorates eventInfo with:
                                       ^
/afs/cern.ch/work/l/leejr/public/TwoLeptonFactory/xAODAnaHelpers/Root/BasicEventSelection.cxx:600:39: note: candidate is:
In file included from /afs/cern.ch/work/l/leejr/public/TwoLeptonFactory/xAODAnaHelpers/xAODAnaHelpers/BasicEventSelection.h:9:0,
                 from /afs/cern.ch/work/l/leejr/public/TwoLeptonFactory/xAODAnaHelpers/Root/BasicEventSelection.cxx:29:
/afs/cern.ch/work/l/leejr/public/TwoLeptonFactory/RootCoreBin/include/PileupReweighting/PileupReweightingTool.h:87:26: note: virtual StatusCode CP::PileupReweightingTool::apply(const EventInfo&, bool)
       virtual StatusCode apply ( const xAOD::EventInfo& eventInfo, bool mu_dependent );
                          ^
/afs/cern.ch/work/l/leejr/public/TwoLeptonFactory/RootCoreBin/include/PileupReweighting/PileupReweightingTool.h:87:26: note:   candidate expects 2 arguments, 1 provided
Compiling IParticleHistsAlgo.o
At global scope:
cc1plus: warning: unrecognized command line option "-Wno-tautological-undefined-compare"
make: *** [/afs/cern.ch/work/l/leejr/public/TwoLeptonFactory/RootCoreBin/obj/x86_64-slc6-gcc49-opt/xAODAnaHelpers/obj/BasicEventSelection.o] Error 1
make: *** Waiting for unfinished jobs....
RootCore: Error failed to compile package xAODAnaHelpers
```

**Comment by rsmith54** [comment]
Hey, give me a few minutes and I can update.

**Comment by rsmith54** [comment]
fixed by #68

**Comment by lawrenceleejr** [comment]
Thanks!
